lib/logzio-nodejs.js

```diff
@@ -113,6 +114,11 @@ LogzioLogger.prototype.log = function(msg) {
     msg = _.assign(msg, this.extraFields);
     msg.type = this.type;
 
+    if (this.addNanoSecs) {
+        var time = process.hrtime();
+        msg['@timestamp_nano_secs'] = time[1];
```

Comment:
@gillyb I think it would help if this field will be a String readable to humans, so I think it should String type like this: "2016-07-20T05:18:35.444Z-701468-283205533" which is lexicographically sortable. 

Here's how to get it:
```
var hrtime = process.hrtime()
var seconds = hrtime[0]
var ns = hrtime[1]
console.log (new Date().toISOString() + "-" + seconds + "-" + ns)
```

Since hrtime() returns the number of seconds (and nanoseconds) since an *arbitrary* point in time (when I ran it, it was July 12th 2016, 05:27:27), we can't create a timestamp out of it. 

So maybe we can call this field "@timestamp_nano_sortable" ?